oak_sign/src/lib.rs

```diff
@@ -116,14 +143,15 @@ impl SignatureBundle {
     pub fn create(input: &[u8], key_pair: &KeyPair) -> anyhow::Result<SignatureBundle> {
         let hash = get_sha256(input);
         Ok(SignatureBundle {
-            public_key: key_pair.pkcs8_public_key(),
+            public_key: key_pair.pkix_public_key(),
```

Comment:
Do we need to internally work with PKIX format? Or the raw public key?

Reply:
Clarified, PTAL.